samples/sample_common/src/sample_utils.cpp

```diff
@@ -930,9 +930,14 @@ mfxStatus GetChromaSize(const mfxFrameInfo & pInfo, mfxU32 & ChromaW, mfxU32 & C
     case MFX_FOURCC_P010:
     case MFX_FOURCC_P210:
     case MFX_FOURCC_P016:
+    case MFX_FOURCC_Y210:
+    case MFX_FOURCC_Y216:
     {
         ChromaW = (pInfo.CropW % 2) ? (pInfo.CropW + 1) : pInfo.CropW;
-        ChromaH = pInfo.FourCC == MFX_FOURCC_P210 ? (mfxU32)pInfo.CropH : (mfxU32)(pInfo.CropH + 1) / 2;
+        ChromaH = (pInfo.FourCC == MFX_FOURCC_P210 ||
```

Comment:
I suggest putting these three as separate code block under this switch (separate current one to two)

Reply:
Thanks for the comment, I updated the patch, please review it again.